README.md

```diff
@@ -1,2 +1,29 @@
-# openmrs-frontend-spa
-The frontend SPA for OpenMRS code
+# openmrs-module-spa
+
+This project is an OpenMRS java module that makes it possible for Tomcat to serve an SPA instead of expecting all routes
+to be handled by server side JSPs/GSPs.
+
+It is part of the microfrotends architecture that is explained at https://github.com/openmrs/openmrs-rfc-frontend.
+
+## Development instructions
```

Comment:
Don't worry -- in the future, we won't have to follow all of the steps in here every time we want to write frontend code.

Once we get a couple key things in place, we'll be able to boot up a webpack-dev-server, create-react-app, `ng serve`, etc and not even have to bother with booting up a backend server because we'll override a deployed environment to point to our localhost for the js assets we're working on. For more info on how that will work, check out https://github.com/joeldenning/import-map-overrides/ and http://single-spa-playground.org/playground/html-file